common/junit-platform-native/src/main/java/org/graalvm/junit/platform/config/jupiter/JupiterConfigProvider.java

```diff
@@ -84,7 +84,10 @@ public void onLoad(NativeImageConfiguration config) {
                 "org.junit.jupiter.params.provider.EnumSource$Mode",
                 // new in Junit 5.10
                 "org.junit.platform.launcher.core.LauncherConfig",
-                "org.junit.jupiter.engine.config.InstantiatingConfigurationParameterConverter"
+                "org.junit.jupiter.engine.config.InstantiatingConfigurationParameterConverter",
+                // new in Junit 5.13?
+                "org.junit.jupiter.api.DisplayNameGenerator$IndicativeSentences",
+                "org.junit.jupiter.engine.discovery.ClassSelectorResolver"
```

Comment:
These are also temporary since they have been added to the `native-image.properties` that ship with JUnit in the meantime.